#joint_names: ["arm_1_joint","arm_2_joint","arm_3_joint","arm_4_joint","arm_5_joint","arm_6_joint","arm_7_joint"]

calibration: [[0.05897, -1.62093, 1.61828, -0.60607, 2.32322, -1.44618, -1.08467]]
intrinsic_00: [[0.13771, -1.61106, 1.60106, -0.90347, 2.30275, -1.28409, -0.93367]]
intrinsic_01: [[0.21571, -1.68461, 1.40137, -1.03150, 2.05505, -1.14005, -0.86222]]
intrinsic_02: [[0.12290, -1.59493, 1.69125, -0.96669, 2.20998, -0.73628, -0.75359]]
intrinsic_03: [[0.04145, -1.63299, 1.55009, -0.74059, 2.33365, -1.81305, -0.93408]]
intrinsic_04: [[0.09348, -1.47456, 1.84216, -0.81108, 2.57727, -1.52091, -1.08165]]
intrinsic_05: [[0.35506, -1.94431, 1.30588, -1.52874, 2.02964, -0.55206, -0.36549]]
intrinsic_06: [[0.35804, -1.59560, 1.56032, -1.30521, 2.62008, -1.53075, -0.90726]]
intrinsic_07: [[-0.10413, -1.48145, 1.93159, -0.36724, 2.62225, -0.86223, -1.27595]]
intrinsic_08: [[-0.14182, -1.53227, 1.83254, -0.29490, 2.62950, -1.86769, -1.23319]]
intrinsic_09: [[0.15222, -1.51624, 1.17735, -1.05862, 2.04233, -0.84345, -0.86415]]
intrinsic_10: [[-0.16216, -1.53930, 1.58198, -0.77218, 2.23807, -0.57952, -0.77109]]
intrinsic_11: [[0.11950, -1.77789, 1.72107, -0.77208, 2.35954, -0.95062, -0.79883]]
intrinsic_12: [[0.47810, -1.70765, 1.47337, -1.02832, 2.10556, -1.29333, -1.09390]]

robot_back_00: [[0.27858, -1.91426, 1.67728, -0.51604, 2.23262, -1.37153, -1.08513]]
robot_back_01: [[0.28562, -1.84651, 1.85173, -0.51519, 2.42707, -1.33735, -0.98657]]
robot_back_02: [[0.14278, -1.94383, 1.54505, -0.28677, 2.14646, -1.77164, -1.29730]]

robot_center_00: [[0.13731, -1.64571, 1.52846, -0.90353, 2.38250, -1.25844, -0.95943]]
robot_center_01: [[0.20687, -1.72509, 1.32585, -1.03147, 2.13898, -1.10319, -0.88570]]
robot_center_02: [[0.12584, -1.63168, 1.61993, -0.96679, 2.30797, -0.70686, -0.78963]]
robot_center_03: [[0.03978, -1.65972, 1.47995, -0.74063, 2.39879, -1.79367, -0.95408]]
robot_center_04: [[0.10019, -1.50285, 1.77285, -0.81123, 2.64718, -1.50560, -1.10729]]
robot_center_05: [[0.33598, -2.00016, 1.23574, -1.52870, 2.14147, -0.49957, -0.42406]]
robot_center_06: [[0.30752, -1.63570, 1.60836, -1.32255, 2.64073, -1.49981, -0.82254]]
robot_center_07: [[-0.10556, -1.47772, 1.95350, -0.36718, 2.59752, -0.86439, -1.27137]]
robot_center_08: [[-0.14268, -1.52910, 1.85496, -0.29485, 2.60801, -1.86944, -1.23018]]
robot_center_09: [[0.13320, -1.57320, 1.24413, -1.04726, 2.03930, -0.92745, -0.82879]]
robot_center_10: [[-0.01801, -1.51120, 1.59218, -0.91910, 2.20896, -0.57444, -0.77252]]
robot_center_11: [[0.21631, -1.70976, 1.73471, -0.89947, 2.33026, -0.90820, -0.81132]]
robot_center_12: [[0.39295, -1.73327, 1.46475, -1.02285, 2.11008, -1.27275, -1.01479]]

robot_left_00: [[0.52988, -1.48926, 1.51747, -1.29446, 2.25344, -1.19169, -1.01815]]
robot_left_01: [[0.59450, -1.37613, 1.60835, -1.38871, 2.19587, -0.69058, -0.92847]]
robot_left_02: [[0.53892, -1.77430, 1.43494, -1.26922, 2.13235, -1.18551, -0.94854]]

robot_right_00: [[-0.10030, -1.81607, 1.60989, -0.72405, 2.37469, -1.35843, -0.76786]]
robot_right_01: [[-0.11751, -1.90875, 1.41719, -0.74886, 2.16570, -1.34387, -0.72777]]
robot_right_02: [[0.18651, -1.83280, 1.69620, -0.98437, 2.34120, -0.84797, -0.64954]]

all_intrinsic: ["intrinsic_00", "intrinsic_01", "intrinsic_02", "intrinsic_03", "intrinsic_04", "intrinsic_05", "intrinsic_06", "intrinsic_07", "intrinsic_08", "intrinsic_09", "intrinsic_10", "intrinsic_11", "intrinsic_12"]
all_robot_center: ["robot_center_00", "robot_center_01", "robot_center_02", "robot_center_03", "robot_center_04", "robot_center_05", "robot_center_06", "robot_center_07", "robot_center_08", "robot_center_09", "robot_center_10", "robot_center_11", "robot_center_12"]
all_robot_left: ["robot_left_00", "robot_left_01", "robot_left_02"]
all_robot_right: ["robot_right_00", "robot_right_01", "robot_right_02"]
all_robot_back: ["robot_back_00", "robot_back_01", "robot_back_02"]